### ISSUE TITLE
[enhancement] Add pre/post string

### ISSUE THREAD

**Issue by fabianmossberg** [henriquehbr/svelte-typewriter#41]
It would be nice to be able to add a pre/post string.

Like this:

```svelte
<Typewriter pre="Come and buy" post="really cheap!">
<h1>cars</h1>
<h1>tractors</h1>
<h1>old computers</h1>
</Typewriter>
``` 

Resulting in:
Come and buy cars really cheap!
Come and buy cars tractors cheap!
Come and buy cars old computers cheap!

Only the word inside the `<Typewriter>` would be typed.

**Comment by fabianmossberg** [comment]
Actually ,it would look even nicer if it was done like this:

```svelte
<Typewriter>
<h1 static>Come and buy</h1>
<h1 static>cars</h1>
<h1>tractors</h1>
<h1>old computers</h1>
<h1 static>really cheap!</h1>
</Typewriter>
```

**Comment by henriquehbr** [comment]
This look like a really good idea @fabianmossberg, thanks for your suggestion!

I also prefer more your second example, as it's more flexible in terms of where each element can be positioned, allowing a more fluid layout, i'll add it on the [features board](https://github.com/henriquehbr/svelte-typewriter/projects/2) and start working on it as soon as i can

**Comment by henriquehbr** [comment]
As currently in Svelte there isn't a supported way to retrieve props passed to elements, i've decided to implement this by making use of data-attributes, the syntax would be almost the same, except that it would have a `data-` prefix on the `static` attribute, example below:

```svelte
<Typewriter>
<h1 data-static>Come and buy</h1>
<h1>cars</h1>
<h1>tractors</h1>
<h1>old computers</h1>
<h1 data-static>really cheap!</h1>
</Typewriter>
```

**Comment by henriquehbr** [comment]
@fabianmossberg Released on version [3.0.0-alpha.2](https://github.com/henriquehbr/svelte-typewriter/blob/main/CHANGELOG.md#300-alpha2---2021-11-06)
